### DISCUSSION TITLE
Issue running on a TTGO T-Display CP32

### DISCUSSION THREAD

**Discussion by Cybermcc007** [revoxhere/duino-coin#572] [Q&A]
I am trying to run a miner on a TTGO T-Display ESP32. I get the sketch to compile, I had to comment out the LED code, and load to the TTGO. It connects to the internet and connects to the server but I keep getting a message stating the buffer size is too small.

CORE1 Buffer size is 4
CORE1 Buffer size is too small. Requesting another job.
CORE2 Asking for a new job for user: Cyberdude007

I am relatively new using the ESP32 so any help would be appreciated.
[TTGO Log.txt](https://github.com/revoxhere/duino-coin/files/6604971/TTGO.Log.txt)

**Comment by revoxhere** [comment]
I suspect that the feedback message got into the job at some point and it doesn't know when to stop reading. Not really sure what to do about it though, as I don't own an ESP32

**Comment by SARMURAI** [comment]
With the latest version 4.2 - it works fine with ESP TTGO T-Display, but no output on LCD due to different Display Driver (TFT_eSPI.h)
